impedance/models/circuits/elements.py

```diff
@@ -61,13 +58,20 @@ def p(parallel):
         Z = \\frac{1}{\\frac{1}{Z_1} + \\frac{1}{Z_2} + ... + \\frac{1}{Z_n}}
 
      """
-    z = len(parallel[0])*[0 + 0*1j]
+    z = len(parallel[0]) * [0 + 0 * 1j]
     for elem in parallel:
         z += 1/elem
     return 1/z
 
 
-@element_metadata(num_params=1, units=['Ohm'])
+# manually add parallel and series operators to circuit elements dict w/o metadata
```

Comment:
This one is also a little long it looks like

Reply:
should be fixed